src/index.js

```diff
@@ -0,0 +1,8 @@
+import './style.css';
+import arrTasks from './modules/tasksList.js';
+import compileTaskItem from './modules/taskItem.js';
+
+const ulTasks = document.getElementById('ul-tasks');
+for (let i = 0; i < arrTasks.length; i += 1) {
+  ulTasks.appendChild(compileTaskItem(arrTasks[i]));
+}
```

Comment:
- [OPTIONAL] I suggest using higher-order functions such as `map` it will make the code cleaner and easy to understand 👍🏼 consider the example below:

```javascript
arrTasks.map((task) => ulTasks.appendChild(compileTaskItem(task)));
```